spectral_cube/tests/test_spectral_cube.py

```diff
@@ -344,6 +342,36 @@ def test_apply_everywhere(self, operation, value, data_advs, use_dask):
         # allclose fails on identical data?
         #assert_allclose(d1o, c1o.filled_data[:])
 
+
+    @pytest.mark.parametrize(('operation', 'value'), ((operator.add, 0.5*u.K),
+                                                      (operator.sub, 0.5*u.K),))
+    def test_apply_everywhere_plusminus(self, operation, value, data_advs, use_dask):
+        c1, d1 = cube_and_raw(data_advs, use_dask=use_dask)
+
+        assert c1.unit == value.unit
+
+        # append 'o' to indicate that it has been operated on
+        # value.value: the __add__ function explicitly drops the units
+        c1o = c1._apply_everywhere(operation, value.value, check_units=False)
+        d1o = operation(u.Quantity(d1, u.K), value)
+        assert c1o.unit == c1.unit
+        assert c1o.unit == value.unit
+
+        assert np.all(d1o == c1o.filled_data[:])
```

Comment:
not needed above, but here?
```suggestion
        assert np.all(d1o == c1o.filled_data[:])
        
        del c1
        del c1o
```